### PR TITLE
Update optimizer-hints.md

### PR DESCRIPTION
**Pull request by sultan8252**
Use READ_FROM_STORAGE to display the statement using schema_name

If you need to prompt that the table used by the optimizer is not in the same database, you need to display the statement SCHEMA_NAME

### What is changed, added or deleted? (Required)

Add note information to help users use the hint function

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: optimizer-hints.md [hunk 1]
```diff
@@ -197,6 +197,12 @@ SELECT /*+ AGG_TO_COP() */ sum(t1.a) FROM t t1;
 SELECT /*+ READ_FROM_STORAGE(TIFLASH[t1], TIKV[t2]) */ t1.a FROM t t1, t t2 WHERE t1.a = t2.a;
 ```
 
+> **注意：**
+>
+> 如果需要提示优化器使用的表不在同一个数据库内，需要显示声明 SCHEMA_NAME 例如	> 如果需要提示优化器使用的表不在同一个数据库内，需要显示声明 SCHEMA_NAME 例如
+> `SELECT /*+ READ_FROM_STORAGE(TIFLASH[test1.t1,test2.t2]) */ t1.a FROM test1.t t1, test2.t t2 WHERE t1.a = t2.a;`	
+
+
```
Comment (TomShawn) [inline optimizer-hints.md:205]:
```suggestion
> **注意：**
>
> 如果需要提示优化器使用的表不在同一个数据库内，需要显示声明 `SCHEMA_NAME`。例如 `SELECT /*+ READ_FROM_STORAGE(TIFLASH[test1.t1,test2.t2]) */ t1.a FROM test1.t t1, test2.t t2 WHERE t1.a = t2.a;`。

```

**Comment by TomShawn** [comment]
@sultan8252 Thanks for your contribution. Would you specify all the applicable versions by ticking the boxes in the description part?

**Review by lzmhhh123 — COMMENTED** [review]
LGTM

File: optimizer-hints.md [hunk 2]
```diff
@@ -197,6 +197,10 @@ SELECT /*+ AGG_TO_COP() */ sum(t1.a) FROM t t1;
 SELECT /*+ READ_FROM_STORAGE(TIFLASH[t1], TIKV[t2]) */ t1.a FROM t t1, t t2 WHERE t1.a = t2.a;
 ```
 
+> **注意：**
+>
+> 如果需要提示优化器使用的表不在同一个数据库内，需要显示声明 SCHEMA_NAME 例如 `SELECT /*+ READ_FROM_STORAGE(TIFLASH[test1.t1,test2.t2]) */ t1.a FROM test1.t t1, test2.t t2 WHERE t1.a = t2.a;`
```
Comment (TomShawn) [inline optimizer-hints.md:202]:
```suggestion
> 如果需要提示优化器使用的表不在同一个数据库内，需要显示声明 `SCHEMA_NAME`。例如 `SELECT /*+ READ_FROM_STORAGE(TIFLASH[test1.t1,test2.t2]) */ t1.a FROM test1.t t1, test2.t t2 WHERE t1.a = t2.a;`。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**